### DISCUSSION TITLE
Locked out of Copilot

### DISCUSSION THREAD

**Discussion by Loidbae** [community/community#14837] [Copilot Conversations] [answered]
Hello,
I received an invitation for copilot a while ago and I finally came around to test it. As I was installing it it asked me for a personal auth token. I put in one of my tokens and it said there was an error. Then i tried to generate a new token to authenticate with it. But then when I put it in I got locked out, saying 

[INFO] [auth] [2022-04-13T05:41:28.626Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2022-04-13T05:41:28.629Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

I got invited see the Pictures. I got the Email and on the site to join the waitlist, says it's already enabled on my account.

What is going on?

I'm using VsCodium(v166.2) and the extension. (Installed via VSIX)
Windows 10 Pro (64bit)
no vpn
the firewall also doesn't interfere with anything github related.

uninstalling and reinstalling didn't help either. It doesn't event prompt me the login again after that, it straight up gives me that error message.

also https://github.com/github/feedback/discussions/14239 didnt't work because there is no github-copilot folder for me...

![email](https://user-images.githubusercontent.com/31895243/163109340-d4256de9-e193-4782-9891-f11f215d28ba.PNG)
![onPage](https://user-images.githubusercontent.com/31895243/163109342-39095d0a-0123-4000-8881-211fc5e6ccd4.PNG)

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by Loidbae** [reply]
Okay, I logged out and was able to get the copilot login prompt again. But no matter what personal auth token I use it says I'm not authorized to use it. Same error message like the one in my initial post.

**Reply by davecheney** [reply]
Oh, i just saw you're using VSCodum.

**Comment by davecheney** [comment, marked as answer]
I'm sorry, Copilot only works with VSCode, VSCodium is not supported at the moment.

**Reply by Loidbae** [reply]
Oh yeah I just installed VsCode and it works perfectly now. VsCodium doesn't seem to have the api set up for it. Thanks a bunch!

**Comment by Crypto-Cat** [comment]
Very disappointing. Enabling Microsoft tracking shouldn't be a requirement to use Copilot!
